build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
Perhaps use the `script` directory instead of putting this in project root?

Reply:
I'm trying to install conan from within CmakeList.txt which seems to work locally but not on GitHub Actions.
It fails with the following:

  catch2/2.13.7: Downloaded recipe revision 0
  ERROR: Not able to automatically detect '/usr/bin/cc' version
  ERROR: Unable to find a working compiler

I've tried different ways of setting compilers and version without success.

Ideas?

PS. Can recommend audio books instead of building ERT on Bergensbanen.

Reply:
I'm listening to the Dune audiobook and I just need _long_ breaks between each 5 minute segment. Interesting premise, questionable execution.